libs/utils/executor.py

```diff
@@ -272,13 +273,34 @@ def _setup_group(self, tc, group):
         # Configure each required attribute
         group.set(**tc['cgroups']['conf'][kind][name])
 
+    def _setup_files(self, tc):
+        if 'files' not in tc:
+            logging.debug('%14s - Configuration not provided', 'Files')
+            return True
+        for name, value in tc['files'].iteritems():
+            check = True
+            if name.startswith('+/'):
```

Comment:
When I proposed the "+" prefix I was mainly thinking about an "append" feature.

A syntax like this one:
```python
# By default we do not check for file to exists and/or written value
check = False
if name.startswith('!/'):
    # If the filename has a "!" at the beginning, than we want to enforce
    check = True
```
should not be more intuitive?